.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -107,14 +107,10 @@ jobs:
           method: kubeconfig
           kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
 
-      - name: Deploy to K8S
+      - name: Update the corresponding configmap
         run: |
-          echo "=============== Update the corresponding configmap ==============="
-          kubectl create configmap --dry-run=client -o yaml --from-env-file=netmanager/.env env-platform-production | kubectl replace -f - -n production
-          echo "=============== Deploy the application ==============="
-          kubectl apply -f k8s/netmanager/prod-airqo-platform-frontend.yaml
-          kubectl set image deployment/airqo-platform-frontend airqo-platform=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} -n production
-          kubectl annotate deployment/airqo-platform-frontend kubernetes.io/change-cause="Image updated to ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n production
+          kubectl create configmap --dry-run=client -o yaml \
+          --from-env-file=netmanager/.env env-platform-production | kubectl replace -f - -n production
```

Comment:
[Seperate the (continous) deployment from the (continous) integration](https://github.com/airqo-platform/AirQo-frontend/pull/960/commits/758efef275bde76d5f62475ee0a76664e3ce5949)